### ISSUE TITLE
Saving active dhcp config deletes /etc/dnsmasq.d/02-pihole-dhcp.conf

### ISSUE THREAD

**Issue by smallpygmy** [pi-hole/pi-hole#1080]
**In raising this issue, I confirm the following (please check boxes, eg [X]) Failure to fill the template will close your issue:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*
- [x] The issue I'm reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**

_5_

---
**[FEATURE REQUEST | QUESTION | OTHER]:**

Please [submit your feature request here](https://discourse.pi-hole.net/c/feature-requests), so it is votable by the community.  It's also easier for us to track.

**[BUG | ISSUE] Expected Behaviour:**
When enabling DHCP in the settings web page, saving the settings should write them to the file /etc/dnsmasq.d/02-pihole-dhcp.conf

**[BUG | ISSUE] Actual Behaviour:**
the file /etc/dnsmasq.d/02-pihole-dhcp.conf is deleted instead of being updated with the new values

**[BUG | ISSUE] Steps to reproduce:**

Ensure that the file /etc/dnsmasq.d/02-pihole-dhcp.conf exists on the file system
From the settings page, enable the DHCP server and click save.
Check if the file still exists on the file system.

**The bug is located in the file webpage.sh at line 184 (source "${setupVars}").  This line should be outside of this if statement (i.e. it should be on line 181 and not 184).**
The issue is that the dhcp enabled setting is being evaluated before the settings have been read from the file.

**(Optional) Debug token generated by `pihole -d`:**

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

**Comment by DL6ER** [comment]
Meanwhile you can run the following one-liner to circumvent the issue:

```
source /etc/pihole/setupVars.conf; source /etc/.pihole/advanced/Scripts/webpage.sh; ProcessDHCPSettings
```

**Comment by DL6ER** [comment]
Already fixed in v2.11.1
